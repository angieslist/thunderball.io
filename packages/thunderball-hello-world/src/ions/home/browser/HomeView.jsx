import React from 'react';
import Helmet from 'react-helmet';
import { compose, pure } from 'recompose';
import './home.scss';

// This must be a 'class' and not a pure render method so that HMR will work
/* eslint react/prefer-stateless-function: 0 */
class HomeView extends React.Component {
  render() {
    return (
      <div>
        <Helmet title="Thunderball hello world" />
        <h1>Welcome to Thunderball hello world.</h1>
      </div>
    );
  }
}

HomeView.propTypes = {
};

export default compose(
  pure,
)(HomeView);
