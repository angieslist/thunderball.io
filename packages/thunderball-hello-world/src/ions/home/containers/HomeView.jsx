import React from 'react';
import { hot } from 'react-hot-loader';
import Helmet from 'react-helmet-async';
import Home from '../components/Home';

const HomeView = () => (
  <div>
    <Helmet title="Home" />
    <Home />
  </div>
);

export default hot(module)(HomeView);
