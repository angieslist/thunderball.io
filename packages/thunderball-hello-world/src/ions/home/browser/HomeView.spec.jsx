import React from 'react';
import { render } from 'enzyme';
import toJson from 'enzyme-to-json';
import HomeView from './HomeView';

describe('<HomeView />', () => {
  it('renders', () => {
    const wrapper = render(<HomeView />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
