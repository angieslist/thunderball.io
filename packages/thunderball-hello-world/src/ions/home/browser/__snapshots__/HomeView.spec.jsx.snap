// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeView /> renders 1`] = `
<div>
  <h1>
    Welcome to Thunderball hello world.
  </h1>
  <div>
    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur vehicula laoreet sem, eget blandit eros dapibus id. Nam et metus nec elit placerat molestie. Etiam ultricies semper finibus. Maecenas efficitur risus elit. Sed ac felis vulputate, ornare neque ut, tempor dui. Sed efficitur auctor est quis faucibus. Quisque venenatis ornare tempus. Integer odio odio, fringilla vitae urna et, dapibus posuere felis. Aenean tristique semper erat, ac finibus augue pharetra eget. Mauris efficitur turpis vitae ante suscipit molestie.
  </div>
</div>
`;
